src/devices/lupus.ts

```diff
@@ -49,13 +49,15 @@ const definitions: Definition[] = [
         vendor: 'Lupus',
         description: '1 channel relay',
         extend: [onOff()],
```

Comment:
```suggestion
        extend: [onOff({powerOnBehavior: false})],
```

To fix the power on behaviour error